src/unix/linux_like/linux/mod.rs

```diff
@@ -920,6 +986,34 @@ cfg_if! {
     }
 }
 
+cfg_if! {
+    if #[cfg(libc_union)] {
+        s_no_extra_traits! {
```

Comment:
I think this doesn't need `cfg_if` since there is no else branch?

```suggestion
#[cfg(libc_union)]
s_no_extra_traits! {
    // ...
```

Reply:
true, although that appears to be true for other declarations too (e.g. the `extra_traits` one right below), and the cfg_if right above has this too. I'm happy to change it of course, but I went with consistency. 

So, do you want to see this changed just here (and be inconsistent with the other definitions surrounding it)?

Reply:
Ah yeah good point, I guess you may as well leave it. I'll do a cleanup pass at some point...